### PR TITLE
EMBCESSMOD-3312: Paper Based Search Results changes

### PR DESCRIPTION
**Pull request by stairaku**

### CODE REVIEW

File: responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.component.ts [hunk 1]
```diff
@@ -105,15 +119,58 @@ export class EvacueeSearchResultsComponent implements OnInit {
   }
 
   openWizard(): void {
-    this.cacheService.set(
-      'wizardOpenedFrom',
-      '/responder-access/search/evacuee'
-    );
-    this.evacueeSessionService.setWizardType(WizardType.NewRegistration);
+    if (this.evacueeSessionService.paperBased === true) {
```
Comment (SodhiA1) [inline responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.component.ts:122]: remove === true from condition, boolean resolves to true/false

Comment (stairaku) [reply responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.component.ts]: fixed!

File: responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.service.ts [hunk 2]
```diff
@@ -77,8 +81,16 @@ export class EvacueeSearchResultsService {
       );
   }
 
+  public essFileExists(
+    essFile: string
```
Comment (SodhiA1) [inline responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.service.ts:85]: rename to paperFileId or externalReferenceId

Comment (stairaku) [reply responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.service.ts]: fixed!

File: responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.component.ts [hunk 3]
```diff
@@ -105,15 +119,58 @@ export class EvacueeSearchResultsComponent implements OnInit {
...
+      this.evacueeSearchResultsService
+        .essFileExists(this.paperBasedEssFile)
+        .subscribe({
+          next: (result) => {
```
Comment (SodhiA1) [inline responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.component.ts:126]: Think about restructuring this code. Maybe call the exists api in the page load and hold the value, that will simplify this method

**Review by SodhiA1 — APPROVED** [review]

**Comment by SodhiA1** [comment]
Also remove console.log statements 🗡️

**Pull request merged**